Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.py

```diff
@@ -576,33 +783,123 @@ def riot_command(client: Client, args: Dict) -> CommandResults:
     response = remove_empty_elements(response)
     name = ""
     if response.get("riot") is False or response.get("riot") == "false":
-        name = "Potentially harmful IP"
+        name = "GreyNoise: IP Not Found in RIOT"
+        hr = {
+            "IP": response.get("ip"),
+            "RIOT": response.get("riot"),
+        }
+        headers = ["IP", "RIOT"]
     elif response.get("riot") is True or response.get("riot") == "true":
-        name = "Benign IP"
-    if response.get("logo_url", "") != "":
-        del response["logo_url"]
-    headers = ["IP", "RIOT", "Category", "Name", "Description", "Last Updated"]
-    hr = {
-        "IP": f"[{response.get('ip')}](https://viz.greynoise.io/riot/{response.get('ip')})",
-        "RIOT": response.get("riot"),
-        "Category": response.get("category"),
-        "Name": response.get("name"),
-        "Description": response.get("description"),
-        "Last Updated": response.get("last_updated")
-    }
-    human_readable = tableToMarkdown(
-        name=name, t=hr, headers=headers, removeNull=True
-    )
+        if response.get("logo_url", "") != "":
+            del response["logo_url"]
+        if response.get("trust_level") == "1":
+            response["trust_level"] = "1 - Reasonably Ignore"
+        elif response.get("trust_level") == "2":
+            response["trust_level"] = "2 - Commonly Seen"
+        name = "GreyNoise: IP Belongs to Common Business Service"
+        hr = {
+            "IP": f"[{response.get('ip')}](https://www.greynoise.io/viz/riot/{response.get('ip')})",
+            "Name": response.get("name"),
+            "Category": response.get("category"),
+            "Trust Level": response.get("trust_level"),
+            "Description": response.get("description"),
+            "Last Updated": response.get("last_updated"),
+        }
+        headers = RIOT_HEADERS
+
+    human_readable = tableToMarkdown(name=name, t=hr, headers=headers, removeNull=True)
     return CommandResults(
-        outputs_prefix="GreyNoise.Riot",
+        outputs_prefix="GreyNoise.RIOT",
```

Comment:
This change is breaking BC, are you aware of that?

Reply:
Can you explain why? What would the user impact be other than possibly having both a Riot and RIOT section in context data?  They would both still be under the GreyNoise section and would be easily identifiable?

Reply:
So, it is returned to context data in both the IP and RIOT commands.  Here is a sample output:

![image](https://user-images.githubusercontent.com/38439955/136965888-983a826f-91fc-45fa-9314-112ec0ede22f.png)